packages/patternfly-3/patternfly-react/src/components/CopyToClipboard/CopyToClipboardBlockBar.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Panel } from 'react-bootstrap';
+import { Form, Icon, Button, OverlayTrigger, Tooltip } from '../..';
+
+export const CopyToClipboardBlockBar = props => {
+  const { value, onChange, onCopy, onClick, copied, show } = props;
+  const iconName = copied ? 'check' : 'paste';
+  const tooltip = (
+    <Tooltip id={copied ? 'tooltip-copied' : 'tooltip-copy-to-clipboard'}>
+      {copied ? 'Copied' : 'Copy to Clipboard'}
+    </Tooltip>
+  );
+  return (
+    <Form.FormGroup>
+      <Form.InputGroup>
+        <Form.InputGroup.Button>
+          <Button onClick={onClick}>
+            <Icon type="fa" name={show ? 'angle-down' : 'angle-right'} />
+          </Button>
+        </Form.InputGroup.Button>
+        <Form.FormControl
```

Comment:
This control also seems to be missing an accessible name. In VoiceOver, it does read the full text aloud, but this may not be desirable. For example, if the value was a secret hash of some type, screen readers would begin reading every single character which probably isn't what the user would want. Instead, I think it would be better to use an aria-label to describe briefly what the nature of the text to be copied is, rather than it's actual value. Then, with the panel expanded, the user has the ability to read to full text if they wish.

<img width="1039" alt="screen shot 2018-11-27 at 10 35 48 am" src="https://user-images.githubusercontent.com/5942899/49094130-81d0d180-f233-11e8-9ce9-c3165ce5891f.png">